tests/system/demos/base.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import mlrun
+
+from tests.system.base import TestMLRunSystem
+
+
+class TestDemo(TestMLRunSystem):
+
+    project_name = ''
+
+    def custom_setup(self):
+        mlrun.set_environment(
+            artifact_path='/User/data', project=self.project_name,
+        )
+        self._artifact_path = os.path.join(
+            mlrun.mlconf.artifact_path, 'pipeline/{{workflow.uid}}'
+        )
```

Comment:
Why `artifact_path` in the environment and in `self` are different ?

Reply:
The first is the general one, and the second is run specific, I'll change the name